src/ray/object_manager/object_buffer_pool.cc

```diff
@@ -327,7 +327,10 @@ Status ObjectBufferPool::EnsureBufferExists(const ObjectID &object_id,
 
 void ObjectBufferPool::FreeObjects(const std::vector<ObjectID> &object_ids) {
   absl::MutexLock lock(&pool_mutex_);
-  RAY_CHECK_OK(store_client_->Delete(object_ids));
+  Status s = store_client_->Delete(object_ids);
+  if (!s.ok()) {
+    RAY_LOG(ERROR) << "Failed to delete objects from plasma store (non-fatal): " << s;
```

Comment:
```suggestion
    RAY_LOG(ERROR) << "Failed to delete objects from plasma store: " << s;
```
don't need the non fatal in parens there, and maybe we should say that they will get evicted or spilled under memory pressure so users aren't as worried?